jwala-services/src/main/java/com/cerner/jwala/service/webserver/impl/WebServerServiceImpl.java

```diff
@@ -208,6 +220,7 @@ public boolean isStarted(WebServer webServer) {
     public void updateState(final Identifier<WebServer> id, final WebServerReachableState state, final String errorStatus) {
         webServerPersistenceService.updateState(id, state, errorStatus);
         inMemoryStateManagerService.put(id, state);
+        messagingTemplate.convertAndSend(topicServerStates, new CurrentState<>(id, state, DateTime.now(), StateType.WEB_SERVER));
```

Comment:
The WebServerStateSetterWorker setState sends a message to the UI already via the MessageService. If we put this here, the message will always be sent 2x. Can we put this after generation ? When we set the state to STOPPED ? In addition "updateState" in my opinion should just update the persistence layer and the hasmap, it's up to the client service to do notifications.